pandas/core/strings.py

```diff
@@ -2051,7 +2051,7 @@ def _get_series_list(self, others, ignore_index=False):
                 return ([Series(others, index=idx)], False)
         raise TypeError(err_msg)
 
-    def cat(self, others=None, sep=None, na_rep=None, join=None):
+    def cat(self, others=None, sep='', na_rep=None, join=None):
```

Comment:
there is likely some other check for ``sep is None`` in the codebase, this shouldn't pass, and as I said pls change back to the original modulo the doc-string.